contracts/src/CrossChainApplications/NativeTokenBridge/README.md

```diff
@@ -0,0 +1,34 @@
+# Native-to-Native Token Bridge
+
+A pair of smart contracts built on top of Teleporter to support using an ERC20 token or the native token of any `subnet-evm` chain as the native token for a given subnet.
+
+## Design
+The native-to-native bridge is implemented using two primary contracts.
```

Comment:
Can you make this a ### subsection, and add another ### subsection for the ERC20-native bridge? That can be minimal, but we should include it for completeness.

Reply:
When you do so, can you make it clear that a `NativeTokenDestination` can only be paired with exactly one instance of `NativeTokenSource` OR `ERC20TokenSource`? I could see some confusion where a reader may think a `NativeTokenDestination` can be paired with one each of `NativeTokenSource` and `ERC20TokenSource`.